lib/insights/api/common/filter.rb

```diff
@@ -14,21 +14,25 @@ class Filter
         #   An AR model that acts as the base collection to be filtered
         # raw_filter::
         #   The filter from the request query string
-        # api_doc_definition::
-        #   The documented object definition from the OpenAPI doc
+        # api_doc_definitions::
+        #   The documented object definitions from the OpenAPI doc
```

Comment:
How man definitions are there?  Isn't this the definition of a single object (e.g. Cluster)?

Reply:
Also, this breaks API compatability, so it requires a major version.

Reply:
sorry, I had that earlier, but is no longer the case, I'll update the comment.